eegnb/experiments/auditory_ssaep/ssaep_onefreq.py

```diff
@@ -0,0 +1,152 @@
+"""
+Generate Steady-State Auditory Evoked Potential (SSAEP)
+=======================================================
+
+Steady-State Auditory Evoked Potential (SSAEP) - also known as Auditory
+Steady-State Response (ASSR) - stimulus presentation.
+
+This variant allows for a single tone (single pair of carrier and modulation frequency)
+
+"""
+
+from time import time, sleep
+from optparse import OptionParser
+
+import numpy as np
+from pandas import DataFrame
+from psychopy import prefs
+prefs.general['audioLib'] = ['pygame']
+from psychopy import visual, core, event, sound
+from pylsl import StreamInfo, StreamOutlet
+from scipy import stats
+
+import os
+from glob import glob
+from random import choice
+
+
+from eegnb import generate_save_fn
+
+
+def present(duration=120, eeg=None, save_fn=None, iti = 0.5, soa = 3.0, jitter = 0.2, 
+            n_trials = 150, cf1 = 1000, amf1 = 40):
+
+    # Create markers stream outlet
+    info = StreamInfo('Markers', 'Markers', 1, 0, 'int32', 'myuidw43536')
+    outlet = StreamOutlet(info)
+
+    markernames = [1]
```

Comment:
this was originally an array of two values, I just removed one of them since I only have one frequency